test/framework/config.go

```diff
@@ -258,6 +266,7 @@ var defaultConf = E2eConfig{
 	ZoneIngressApp:       "kuma-ingress",
 	UniversalE2ELogsPath: path.Join(os.TempDir(), "e2e"),
 	CleanupLogsOnSuccess: false,
+	KumaDeltaKDS:         false,
```

Comment:
this is automatically set by the env var?

Reply:
Yes, it's overridden by env